rootfs/api/migrations/0015_auto_20160822_2103.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.8 on 2016-08-22 21:03
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('api', '0013_auto_20160816_2122'),
```

Comment:
Doesn't this also need to depend on 0014? Suppose the migration system could be smart enough to resolve things